.gitignore

```diff
@@ -1,7 +1,8 @@
+.gitignore
 dsn.txt
 CR.txt
 names.json
 config.json*
 *.gob
 *.exe
-fryatog
\ No newline at end of file
+Fryatog
```

Comment:
But why

Reply:
But why not.

Reply:
Turns out gitignore accepts patterns so we can do [fF]ryatog

Reply:
Oh hey! I... did not know that. Good information.

Reply:
Sooooo..can update?